addons/web_graph/static/src/js/graph_view.js

```diff
@@ -83,6 +83,9 @@ instance.web_graph.GraphView = instance.web.View.extend({
             groupbys = this.get_groupbys_from_searchview(),
             col_group_by = groupbys.col_group_by,
             measures = groupbys.measures;
+            if(measures.length === 0 && context.measures) {
```

Comment:
bad indentation: should be 1 tab on the left (sorry for being annoying, but it is confusing)